### ISSUE TITLE
last-release>> error on website

### ISSUE THREAD

**Issue by erg** [slavapestov/factor#1432]
```
[2015-08-01T23:02:37.419545Z] NOTICE handle-client: { T{ duplex-stream f ~decoder~ ~encoder~ } T{ inet4 f "66.249.65.214" 42657 } T{ inet4 f "0.0.0.0" 8080 } }
[2015-08-01T23:02:37.419901Z] NOTICE accepted-connection: remote: 66.249.65.214:42657, local: 0.0.0.0:8080
[2015-08-01T23:02:37.423507Z] NOTICE httpd-hit: { "GET" URL" http://builds.factorcode.org/release?cpu=x86.32&os=openbsd" }
[2015-08-01T23:02:37.423909Z] NOTICE httpd-header: { "user-agent" "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)" }
[2015-08-01T23:02:37.424176Z] NOTICE httpd-header: { "x-forwarded-for" f }
[2015-08-01T23:02:37.432119Z] DEBUG init-user: f
[2015-08-01T23:02:37.434658Z] ERROR do-request: Generic word last-release>> does not define a method for the POSTPONE: f class.
[--------------------] ERROR do-request: Dispatching on object: f
```

**Comment by erg** [comment]
Some more, ``last-release>>`` ``last-report>>`` ``os>>`` ``title>>``

```
[2015-08-01T11:43:23.337388Z] NOTICE handle-client: { T{ duplex-stream f ~decoder~ ~encoder~ } T{ inet4 f "173.236.169.235" 50559 } T{ inet4 f "0.0.0.0" 8080 } }
[2015-08-01T11:43:23.337748Z] NOTICE accepted-connection: remote: 173.236.169.235:50559, local: 0.0.0.0:8080
[2015-08-01T11:43:23.339884Z] NOTICE httpd-hit: { "GET" URL" http://builds.factorcode.org/report?cpu=x86.32&os=winnt" }
[2015-08-01T11:43:23.340237Z] NOTICE httpd-header: { "user-agent" f }
[2015-08-01T11:43:23.340481Z] NOTICE httpd-header: { "x-forwarded-for" f }
[2015-08-01T11:43:23.348137Z] DEBUG init-user: f
[2015-08-01T11:43:23.350438Z] ERROR do-request: Generic word last-report>> does not define a method for the POSTPONE: f class.
[--------------------] ERROR do-request: Dispatching on object: f



[2015-08-04T13:40:48.260276Z] NOTICE handle-client: { T{ duplex-stream f ~decoder~ ~encoder~ } T{ inet4 f "66.249.65.214" 55527 } T{ inet4 f "0.0.0.0" 8080 } }
[2015-08-04T13:40:48.260639Z] NOTICE accepted-connection: remote: 66.249.65.214:55527, local: 0.0.0.0:8080
[2015-08-04T13:40:48.264109Z] NOTICE httpd-hit: { "GET" URL" http://builds.factorcode.org/package?cpu=x86.64&os=openbsd" }
[2015-08-04T13:40:48.264497Z] NOTICE httpd-header: { "user-agent" "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)" }
[2015-08-04T13:40:48.264741Z] NOTICE httpd-header: { "x-forwarded-for" f }
[2015-08-04T13:40:48.272413Z] DEBUG init-user: f
[2015-08-04T13:40:48.274639Z] ERROR do-request: Generic word os>> does not define a method for the POSTPONE: f class.
[--------------------] ERROR do-request: Dispatching on object: f


[2015-07-31T10:37:55.128657Z] NOTICE handle-client: { T{ duplex-stream f ~decoder~ ~encoder~ } T{ secure f ~inet4~ } T{ secure f ~inet4~ } }
[2015-07-31T10:37:55.129022Z] NOTICE accepted-connection: remote: 66.249.65.141:47876 (secure), local: 0.0.0.0:8431 (secure)
[2015-07-31T10:37:55.158792Z] NOTICE httpd-hit: { "GET" URL" https://www.concatenative.org/wiki/diff?__s=4691632185702894941&new-id=2268&__u=https%3A%2F%2Fwww.concatenative.org%2Fwiki%2Frevisions%2FFactor%2FBug%2520reports%2FUI%2520bugs&old-id=586" }
[2015-07-31T10:37:55.159253Z] NOTICE httpd-header: { "user-agent" "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)" }
[2015-07-31T10:37:55.159498Z] NOTICE httpd-header: { "x-forwarded-for" f }
[2015-07-31T10:37:55.168283Z] DEBUG init-user: f
[2015-07-31T10:37:55.169568Z] ERROR do-request: Generic word title>> does not define a method for the POSTPONE: f class.
[--------------------] ERROR do-request: Dispatching on object: f
```

**Comment by mrjbq7** [comment]
``current-builder`` returns ``f`` if ``select-tuple`` isn't able to lookup a builder with the specified ``os`` and ``cpu``... looks like the URL is for "openbsd-x86.64" which hasn't been valid for awhile... maybe we should handle that case better.

**Comment by mrjbq7** [comment]
Also "winnt-x86-32" and "openbsd-x86-32" so at least it's easy to test.

**Comment by mrjbq7** [comment]
Fixed in a9f842430291384fe8b6ce3718610261c927729a.
